Dockerfile

```diff
@@ -23,7 +23,8 @@ COPY internal/controller/ internal/controller/
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
 RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager cmd/main.go
 
-FROM ubi8:8.7-929
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1123
+RUN yum update -y && yum upgrade -y
```

Comment:
Usually whenever there are major CVEs then base image with pointing to latest will have the fixes, this will add unnecessary layer with additional cache and files which is not required. Just adding `:latest` tag and building this image often should be good enough.

Reply:
Without upgrading the packages, [rsct-operator:0.0.2](https://quay.io/repository/vahirwad/rsct-operator/manifest/sha256:715c2ae448921f38358a2fb26e7d4e795d64ca08c402870392bbaaa521ef8c40?tab=vulnerabilities&fixable=true) image is having 2 High-level vulnerabilities.

Reply:
yes, that's right.. these reported CVEs got identified couple of days back only, this can even happen post publishing the image as well. The best way to avoid such situation is to ship as less as possible in the container image(e.g: in this case python is the trouble maker, hence using ubi9/ubi-minimal will be a better idea if you are not using any of those other packages!)

Reply:
Thanks, now using `ubi9/ubi-minimal`